executor/src/witgen/machines/machine_extractor.rs

```diff
@@ -267,23 +261,26 @@ impl<'a, T: FieldElement> MachineExtractor<'a, T> {
         &self,
         mut witnesses: HashSet<PolyID>,
         all_witnesses: &HashSet<PolyID>,
-        identities: &[Identity<T>],
     ) -> HashSet<PolyID> {
         loop {
             let count = witnesses.len();
-            for i in identities {
-                match i {
-                    Identity::Polynomial(_) | Identity::BusSend(_) | Identity::BusReceive(_) => {
-                        // Any current witness in the identity adds all other witnesses.
-                        let in_identity = &self.fixed.polynomial_references(i) & all_witnesses;
-                        if in_identity.intersection(&witnesses).next().is_some() {
-                            witnesses.extend(in_identity);
-                        }
-                    }
-                    Identity::Connect(..) => {
```

Comment:
We no longer panic for connect identities, but I think they are handled correctly?